src/packages/core/property-editor/uis/user-group-picker/property-editor-ui-user-group-picker.element.ts

```diff
@@ -0,0 +1,31 @@
+import { html, customElement, property } from '@umbraco-cms/backoffice/external/lit';
+import { UmbTextStyles } from '@umbraco-cms/backoffice/style';
+import { UmbPropertyEditorUiElement } from '@umbraco-cms/backoffice/extension-registry';
+import { UmbLitElement } from '@umbraco-cms/internal/lit-element';
+import { UmbPropertyEditorConfigCollection } from '@umbraco-cms/backoffice/property-editor';
+
+/**
+ * @element umb-property-editor-ui-user-group-picker
+ */
+@customElement('umb-property-editor-ui-user-group-picker')
+export class UmbPropertyEditorUIUserGroupPickerElement extends UmbLitElement implements UmbPropertyEditorUiElement {
+	@property()
+	value = '';
+
+	@property({ attribute: false })
+	public config?: UmbPropertyEditorConfigCollection;
+
+	render() {
+		return html`<div>umb-property-editor-ui-user-group-picker</div>`;
```

Comment:
The [`<umb-user-group-input>`](https://github.com/umbraco/Umbraco.CMS.Backoffice/blob/v14.0.0--preview004/src/packages/user/user-group/components/input-user-group/user-group-input.element.ts) component could be wired up here.
@bjarnef Would you be up for continuing work on this PR? 😃

Reply:
@leekelleher yeah, it was basically a copy of `member-group-picker` here:
https://github.com/umbraco/Umbraco.CMS.Backoffice/blob/9b636d40455f1a015b7de7fe373d063fe11e7ee2/src/packages/core/property-editor/uis/member-group-picker/property-editor-ui-member-group-picker.element.ts#L19

Otherwise feel free to push a change to this PR 😄

Reply:
I have resolved the conflicts and updated to use `<umb-user-group-input>` here instead.